src/panels/developer-tools/service/developer-tools-service.ts

```diff
@@ -292,34 +294,58 @@ class HaPanelDevService extends LitElement {
     fields.filter((field) => !field.selector)
   );
 
-  private _isValid = memoizeOne((serviceData, fields, target): boolean => {
-    if (!serviceData?.service) {
-      return false;
-    }
-    const domain = computeDomain(serviceData.service);
-    const service = computeObjectId(serviceData.service);
-    if (!domain || !service) {
-      return false;
-    }
-    if (
-      target &&
-      !serviceData.target &&
-      !serviceData.data?.entity_id &&
-      !serviceData.data?.device_id &&
-      !serviceData.data?.area_id
-    ) {
-      return false;
-    }
-    for (const field of fields) {
+  private _isValid = memoizeOne(
```

Comment:
Since we no longer use this in render, I don't think it is needed to memoize this?

Reply:
done